packages/xo-web/src/xo-app/hub/recipes/recipe-form.js

```diff
@@ -129,6 +139,21 @@ export default decorate([
         (_, { value }) =>
         sr =>
           sr.$pool === get(() => value.pool.id) && isSrWritable(sr),
+      versionList: async () => {
+        const res = await fetch('https://api.github.com/repos/kubernetes/kubernetes/releases')
+        if (res.ok) {
+          const rawList = await res.json()
+          const versionList = rawList
+            .filter(version => !version.tag_name.includes('-'))
+            .map(({ tag_name }) => ({
+              label: tag_name,
+              value: tag_name.slice(1) + '-00',
```

Comment:
What's the `-00` for?

Reply:
When you look at the list of the available packages, you see that the suffix ```-00``` is added to the number version:
```bash
debian@cp-1:~$ apt list -a kubeadm
Listing... Done
kubeadm/kubernetes-xenial,now 1.27.3-00 amd64 [installed]
kubeadm/kubernetes-xenial 1.27.2-00 amd64
kubeadm/kubernetes-xenial 1.27.1-00 amd64
kubeadm/kubernetes-xenial 1.27.0-00 amd64
kubeadm/kubernetes-xenial 1.26.6-00 amd64
kubeadm/kubernetes-xenial 1.26.5-00 amd64
kubeadm/kubernetes-xenial 1.26.4-00 amd64
kubeadm/kubernetes-xenial 1.26.3-00 amd64
kubeadm/kubernetes-xenial 1.26.2-00 amd64
kubeadm/kubernetes-xenial 1.26.1-00 amd64
kubeadm/kubernetes-xenial 1.26.0-00 amd64
kubeadm/kubernetes-xenial 1.25.11-00 amd64
kubeadm/kubernetes-xenial 1.25.10-00 amd64
kubeadm/kubernetes-xenial 1.25.9-00 amd64
kubeadm/kubernetes-xenial 1.25.8-00 amd64
kubeadm/kubernetes-xenial 1.25.7-00 amd64
kubeadm/kubernetes-xenial 1.25.6-00 amd64
kubeadm/kubernetes-xenial 1.25.5-00 amd64
kubeadm/kubernetes-xenial 1.25.4-00 amd64
kubeadm/kubernetes-xenial 1.25.3-00 amd64
kubeadm/kubernetes-xenial 1.25.2-00 amd64
kubeadm/kubernetes-xenial 1.25.1-00 amd64
kubeadm/kubernetes-xenial 1.25.0-00 amd64
kubeadm/kubernetes-xenial 1.24.15-00 amd64
kubeadm/kubernetes-xenial 1.24.14-00 amd64
kubeadm/kubernetes-xenial 1.24.13-00 amd64
kubeadm/kubernetes-xenial 1.24.12-00 amd64
kubeadm/kubernetes-xenial 1.24.11-00 amd64
kubeadm/kubernetes-xenial 1.24.10-00 amd64
kubeadm/kubernetes-xenial 1.24.9-00 amd64
kubeadm/kubernetes-xenial 1.24.8-00 amd64
kubeadm/kubernetes-xenial 1.24.7-00 amd64
kubeadm/kubernetes-xenial 1.24.6-00 amd64
kubeadm/kubernetes-xenial 1.24.5-00 amd64
kubeadm/kubernetes-xenial 1.24.4-00 amd64
kubeadm/kubernetes-xenial 1.24.3-00 amd64
kubeadm/kubernetes-xenial 1.24.2-00 amd64
kubeadm/kubernetes-xenial 1.24.1-00 amd64
kubeadm/kubernetes-xenial 1.24.0-00 amd64
kubeadm/kubernetes-xenial 1.23.17-00 amd64
kubeadm/kubernetes-xenial 1.23.16-00 amd64
kubeadm/kubernetes-xenial 1.23.15-00 amd64
kubeadm/kubernetes-xenial 1.23.14-00 amd64
kubeadm/kubernetes-xenial 1.23.13-00 amd64
kubeadm/kubernetes-xenial 1.23.12-00 amd64
kubeadm/kubernetes-xenial 1.23.11-00 amd64
kubeadm/kubernetes-xenial 1.23.10-00 amd64
kubeadm/kubernetes-xenial 1.23.9-00 amd64
kubeadm/kubernetes-xenial 1.23.8-00 amd64
kubeadm/kubernetes-xenial 1.23.7-00 amd64
kubeadm/kubernetes-xenial 1.23.6-00 amd64
kubeadm/kubernetes-xenial 1.23.5-00 amd64
kubeadm/kubernetes-xenial 1.23.4-00 amd64
```
If that suffix is not set, you won't be able to install the package.

Reply:
Ok, can you add a comment about that? Because it's hard to guess for the next one working on that code :)